plugins/compliance-hub/frontend/public/templates/compliance.html

```diff
@@ -86,7 +86,7 @@
                 <div id="dashboard-purge-graph">
                     <div class="data">
                         <div class="name">{{purges.title}}</div>
-                        <div class="trend {{purges.trend}}">
+                        <div class="trend {{purges.trend}} inverse-trend">
```

Comment:
Why in some places we have additionalStyle in others hardcoded inverse-trend?

Reply:
Ones with "additionalStyle" are parts in #each loops. One item of the array needs to be inverse, and the other doesn't. So, instead of using an if statement -which is the nearly the same in terms of information which will be passed-, I preferred the class to be dynamic.